include/dlaf/matrix/layout_info.h

```diff
@@ -36,7 +36,7 @@ class LayoutInfo {
   /// @pre tile_ld >= 1,
   /// @pre tile_offset_row >= 1,
   /// @pre tile_offset_col >= 1;
-  /// If !size.isEmpty():
+  /// if !size.isEmpty():
```

Comment:
I think it is not going to be rendered as you expect...

<img width="653" alt="image" src="https://user-images.githubusercontent.com/9337627/87937083-4cc26400-ca94-11ea-95ea-eb4b6d7f63bf.png">

Reply:
You're right. Now it should be (slightly) better. What do you think? 

![image](https://user-images.githubusercontent.com/44648667/87941715-d4f83780-ca9b-11ea-9a80-a9c4173ffac6.png)